From 6a670438877eecc6b75407e548dbb51bb849a9ff Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Fri, 22 Nov 2013 16:51:55 -0700
Subject: [PATCH 58/65] Ticket 47599 - Reduce lock scope in retro changelog
 plug-in

Description:  Forgot to add definition of retrocl_cn_lock

https://fedorahosted.org/389/ticket/47599

Reviewed by: richm(Thanks!!)
(cherry picked from commit b330876a1bccd93a8e906ac56a10c002c981ecfc)
(cherry picked from commit f4d5900579c773e5cf4b69eaeba6104078512ab0)
(cherry picked from commit b19239fdca5c00865471acfd5ffc8502c66b914a)
---
 ldap/servers/plugins/retrocl/retrocl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ldap/servers/plugins/retrocl/retrocl.c b/ldap/servers/plugins/retrocl/retrocl.c
index 3e426a7..528434e 100644
--- a/ldap/servers/plugins/retrocl/retrocl.c
+++ b/ldap/servers/plugins/retrocl/retrocl.c
@@ -77,6 +77,7 @@ void* g_plg_identity [PLUGIN_MAX];
 
 Slapi_Backend *retrocl_be_changelog = NULL;
 PRLock *retrocl_internal_lock = NULL;
+Slapi_RWLock *retrocl_cn_lock;
 int retrocl_nattributes = 0;
 char **retrocl_attributes = NULL;
 char **retrocl_aliases = NULL;
-- 
1.8.1.4

